README.md

```diff
@@ -55,15 +55,32 @@ import SEAL from 'node-seal/allows_wasm_node_umd'
 
 #### React-Native
 
-React-native does not support WASM libraries; however, it is possible to run a
-WASM library, including `node-seal` by using a
+React-native does not support Wasm libraries; however, it is possible to run a
+Wasm library, including `node-seal` by using a
 [WebView](https://github.com/react-native-webview/react-native-webview#readme)
 to load both the library and a simple interface to communicate with on top of
 the built-in `postMessage` API. Instead of publicly hosting the web application
 to be rendered by the WebView, it is possible to bundle the mini web application
 into a single HTML file (with JS inlined) and load the HTML file directly to the
 WebView.
 
+#### Cloudflare Workers
+
+The Wasm library needs to be explicitly imported, it will be compiled and
+provided by the Cloudflare Workers runtime. Example:
+
+```javascript
+import SEAL from 'node-seal/throws_wasm_cf_worker_es'
+import wasm from 'node-seal/submodules/SEAL/build/lib/seal_throws_wasm_cf_worker.wasm';
```

Comment:
@s0l0ist  this folder isn't included into the final artifact, should I include this file or copy it somewhere?

Reply:
Hm, I suppose I need to build this and test myself. I haven't tested importing the `wasm` bundle like this in a long time. I'll have to test and let you know.

The place to include it I think would be in the rollup configuration.